cmd/main.go

```diff
@@ -30,13 +30,13 @@ func main() {
 	c, err := cli.New(
 		cli.WithCommandName("kubebuilder"),
 		cli.WithVersion(versionString()),
-		cli.WithDefaultProjectVersion(cfgv3.Version),
 		cli.WithPlugins(
 			&pluginv2.Plugin{},
 			&pluginv3.Plugin{},
+			&declarativev1.Plugin{},
 		),
-		cli.WithDefaultPlugins(cfgv2.Version, &pluginv2.Plugin{}),
-		cli.WithDefaultPlugins(cfgv3.Version, &pluginv3.Plugin{}),
+		cli.WithDefaultPlugins(&pluginv3.Plugin{}),
+		cli.WithDefaultProjectVersion(cfgv3.Version),
```

Comment:
The scenario https://github.com/operator-framework/operator-sdk/pull/4581#discussion_r582216189 needs also be solved in #2019 before we get it merged.

Reply:
I will probably address that scenario in a follow up as it doesn't break the API, it will just add a new `PluginBundle` type that basically groups plugins together. This way, the operator-sdk will be able to create a bundle called `helm/v1` that groups `base.helm/v1`, `manifests/v1` and `scorecard/v1`. This way users will just need to call `operator-sdk init --plugin helm/v1` and kubebuilder will resolve it to the three plugins.

Reply:
If we are able to have a workaround in place in operator-framework/operator-sdk#4581 until there I am totally all in with we do that as follow up. Otherwise, I think we need to have the solution before we get it all merged.

Reply:
This is already solved with `Bundle` and `NewBundle`